commcare_connect/opportunity/tables.py

```diff
@@ -176,15 +176,6 @@ class Meta:
         empty_text = "No user have payments accrued yet."
 
 
-class UserPaymentsTable(tables.Table):
```

Comment:
why don't we need a payments table anymore?

Reply:
This list is now displayed as payment-history pulldown. You can click on 'Last Paid' to see the history in this [staging link](https://connect-staging.dimagi.com/a/nsaxenaccc/opportunity/160/worker_list/?active_tab=payments). Though, it doesn't display the additional columns (payment operator)

There is some out of scope work that displays the rest of the columns as well as part of bulk payments UI.